src/types/Contact.ts

```diff
@@ -24,6 +24,7 @@ export interface Contact {
     is_demo: boolean;
 
     company_name: string;
+    domain: string;
```

Comment:
I believe this property is nullable. 

```suggestion
    domain: string | null;
```